alt1/index.html

```diff
@@ -263,6 +278,9 @@ <h3>Your Roles</h3>
                 <h2>Event History Information</h2>
                 <p>The <strong>Hide Expired</strong> checkbox hides any Expired events.</p>
                 <p>The <strong>Misty Timers</strong> checkbox will change the Time Left column to Misty Timer. If editing the event with this checked, you should input the time that Misty provides.</p>
+                <p>
+                    The <strong>Toggle notifications today</strong> checkbox will <strong>suppress notifications</strong> until 00:00 GMT, after which it resets automatically.
```

Comment:
```suggestion
                    The <strong>Toggle notifications today</strong> checkbox will <strong>suppress notifications</strong> until 00:00 UTC, after which it resets automatically.
```
Forgot to add this in the previous review. GMT right now is ahead 1 hour of UTC (game time)